Issue templates proposal (#3254)
racket/racket: .github/ISSUE_TEMPLATE/bug-report.md

---
name: Bug report

> about: Create a report to help us improve Racket 

Reviewer: ```suggestion
about: Report a problem with Racket or Racket's tools
```

title: ''

> labels: bug

Reviewer: I don't think this needs a label by default.

Reviewer 2: I kinda disagree. Label makes it possible to filter issues. I think we should keep it.

Reviewer: Labels are great, but I'm not sure "bug" is a useful one.

Reviewer 2: IIUC, it's to distinguish bugs from feature requests.

Granted, Racket seems to have bug issues far more than feature request issues.

Author: I've removed the label - everything is a bug if it is not a feature request. Note the current cohort of bug reports don't have a label so this is consistent.
